lib/markdown.js

```diff
@@ -79,7 +79,17 @@ const makeHandler = (visitor) => {
 const handleCode = makeHandler(visitCodeBlock);
 const handleMath = makeHandler(visitMathBlock);
 
-const processor = unified()
+const regularProcessor = unified()
+  .use(markdown)
+  .use(math)
+  .use(handleMath)
+  .use(gfm)
+  .use(remark2rehype, { allowDangerousHtml: true })
```

Comment:
Why is `allowDangerousHtml` set here?

Reply:
remark-raw requires it: https://github.com/rehypejs/rehype-raw/blob/main/readme.md?plain=1#L44-L46